docs/faq.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Frequently Asked Questions'
```

Comment:
@MichaelArestad we use single quotes across the board except for JSX strings. Please update your editor settings -- if we merge this, the next person's edits will just toggle this back and it will dirty up the git history with spurious changes.

Reply:
I think this was me. Sorry folks!

Reply:
Updated